ipv4.go

```diff
@@ -0,0 +1,43 @@
+package ipv4
+
+import (
+	"fmt"
+	"math/rand"
+	"net"
+)
+
+func (nr *Randomizer) GenerateRandomIpv4() net.IP {
```

Comment:
Maybe we can just name this function to `IPv4` and have an optional parameter `cidr string` to allocate the IPv4 from the given cidr range.

```suggestion
func (nr *Randomizer) IPv4() net.IP {
```